scripts/sct_label_vertebrae.py

```diff
@@ -29,15 +29,16 @@
 from spinalcordtoolbox.utils import Metavar, SmartFormatter, ActionCreateFolder, list_type
 import sct_utils as sct
 import sct_straighten_spinalcord
+import spinalcordtoolbox
 
 
 # PARAMETERS
 class Param:
     # The constructor
     def __init__(self):
         self.shift_AP = 32  # 0#32  # shift the centerline towards the spine (in voxel).
-        self.size_AP = 11  # 41#11  # window size in AP direction (=y) (in voxel)
-        self.size_RL = 1  # 1 # window size in RL direction (=x) (in voxel)
+        self.size_AP = 20  # 41#11  # window size in AP direction (=y) (in voxel)
+        self.size_RL = 30  # 1 # window size in RL direction (=x) (in voxel)
```

Comment:
any reason for changing these defaults? that will change behaviour

Reply:
The template has changed, now we have point located on the posterior tip of each disc that you need to see hence the increase in windows size.  I found that these values worked fine.